backend/utils/db.ts

```diff
@@ -0,0 +1,6 @@
+import { PrismaClient } from '@prisma/client';
```

Comment:
same, i think this logic is already in backend/app.ts

Reply:
okay ty i will fix that